src/featurelist.cpp

```diff
@@ -132,43 +188,50 @@ void FeatureList::updateFeatureList(const QByteArray& data) {
   SettingsHolder* settingsHolder = SettingsHolder::instance();
   Q_ASSERT(settingsHolder);
 
-  bool changed = false;
-  QStringList devModeFeatureFlags = settingsHolder->devModeFeatureFlags();
-
   QJsonObject json = QJsonDocument::fromJson(data).object();
-  QJsonValue featuresValue = json["features"];
-  if (!featuresValue.isObject()) {
-    logger.error() << "Error in the json format";
-    return;
-  }
-
-  QJsonObject featuresObj = featuresValue.toObject();
-  for (const QString& key : featuresObj.keys()) {
-    QJsonValue value = featuresObj.value(key);
-    if (!value.isBool()) {
-      logger.error() << "Error in parsing feature enabling:" << key;
-      continue;
+  if (json.contains("featuresOverwrite")) {
```

Comment:
I decided to change the name of the attribute because I don't want to break any custom feature-enabling config.
The new approach is that, if guardian returns a list of features, _those_ must be applied, overwriting everything.

Reply:
I have concerns about this approach that it could make things difficult for us as developers. I could forsee a situation where we need Guardian to disable a feature due to some bug, and we can't turn it on as developers on order to debug it because the contents of `featuresOverwrite` takes precedence over our local settings.

Reply:
Also, I wouldn't worry about changing the name in the JSON, the featurelist is behind a dev menu, so we don't have to feel too bad about potential user breakage here. We're changing the name of the underlying settings anyways, so any previous feature settings aren't going to carry over across an upgrade anyways.